fvm/evm/emulator/config.go

```diff
@@ -13,8 +13,8 @@ import (
 )
 
 var (
-	FlowEVMTestnetChainID = big.NewInt(666)
-	FlowEVMMainnetChainID = big.NewInt(777)
+	FlowEVMTestnetChainID = big.NewInt(6666)
```

Comment:
Already taken: https://github.com/onflow/evm-chains/blob/master/_data/chains/eip155-6666.json

Reply:
hah it was not showing up on the website, any suggested value ?

Reply:
646 and 747 are available